libnavigation-ui/src/main/java/com/mapbox/navigation/ui/map/NavigationMapboxMap.java

```diff
@@ -754,6 +755,20 @@ public void takeScreenshot(NavigationSnapshotReadyCallback navigationSnapshotRea
     mapboxMap.snapshot(navigationSnapshotReadyCallback);
   }
 
+  public void enableVanishingRouteLine() {
+    if (!vanishRouteLineEnabled) {
+      vanishRouteLineEnabled = true;
+      locationComponent.addOnIndicatorPositionChangedListener(indicatorPositionChangedListener);
```

Comment:
Potential Scenario: 
Activity: onCreate -> navigationMapboxMap. enableVanishingRouteLine() -> `locationComponent.addOnIndicatorPositionChangedListener(indicatorPositionChangedListener);` -> Activity: onStart -> navigationMapboxMap internally -> `locationComponent.addOnIndicatorPositionChangedListener(indicatorPositionChangedListener);` (again),
seems that is not expected scenario, right?

Reply:
Good catch. I made an adjustment.